lib/stormpath-sdk/resource/application.rb

```diff
@@ -135,7 +135,9 @@ def api_key_id
     client.data_store.api_key.id
   end
 
-  def create_password_reset_token email
-    password_reset_tokens.create email: email
+  def create_password_reset_token(email, account_store: nil)
```

Comment:
@ecrisostomo Yes, you are right, I've pushed a new version where they can send a resource instance.